tests/spec/unit/create.spec.js

```diff
@@ -49,7 +49,7 @@ function createAndVerify (projectname, projectid) {
     fs.removeSync(tmpDir);
 }
 
-function createAndValidateProjectDirName (projectname, projectid) {
+function createAndValidateProjectDirName (projectname, projectid, copyNodeModules) {
```

Comment:
I would generally prefer something like the following, which makes the meaning of the flag obvious on call sites of the function.

```suggestion
function createAndValidateProjectDirName (projectname, projectid, { copyNodeModules = false } = {}) {
```